### DISCUSSION TITLE
No flame spread during wood combustion

### DISCUSSION THREAD

**Discussion by Eric-Ramboll** [firemodels/fds#12972] [Q&A]
Hi,
I visibly have wood pyrolysis in contact with my &VENT because I have an MLR and flames/smoke on Smokeview.
On the other hand, I don't understand why I don't see a flame spread, nor a temperature rise on my obstruction close to the heat source. I'm wondering if my wooden board really "burns". Can you tell me what is not correctly in my fds file?

[TEST_B01.txt](https://github.com/firemodels/fds/files/15398262/TEST_B01.txt)

**Comment by rmcdermo** [comment]
When I try to run your case, I get this:
```
 Starting FDS ...

 MPI Process      0 started on PN122745

 Reading FDS input file ...

WARNING: SPEC GAZ PYRO is not in the table of pre-defined species. Any unassigned SPEC variables in the input were assigned the properties of nitrogen.
WARNING: MATL AGGLOMERE, REAC 1. No product yields (NUs) set

ERROR(335): SURF PLANCHE cannot have a reacting MATL and IGNITION_TEMPERATURE. (CHID: TEST_B01)

ERROR: FDS was improperly set-up - FDS stopped (CHID: TEST_B01)
```

Please fix this (search user guide on IGNITION_TEMPERATURE) and try your case with the latest code.  Thanks

**Comment by rmcdermo** [comment]
By "this", I mean fix the ERRORs.  The WARNINGs are ok.

**Comment by Eric-Ramboll** [comment]
Hi, Sorry, wrong file.

Here is the file without flame spread.


Sincères salutations

Eric MARCHAND

Ramboll France SAS au capital de 38 115€ RCS Aix-en-Provence 2002 B 1288

SIRET: 443 685 029 00094 APE: 7112B


Information on data protection in accordance with the GDPR<https://brandcentral.ramboll.com/share/Aipwzjt6CGop2o5SaTnE>







Classification: Confidential

De : Randy McDermott ***@***.***>
Envoyé : mercredi 22 mai 2024 13:08
À : firemodels/fds ***@***.***>
Cc : Eric Marchand ***@***.***>; Author ***@***.***>
Objet : Re: [firemodels/fds] No flame spread during wood combustion (Discussion #12972)

Vous ne recevez pas souvent de courriers de la part de ***@***.******@***.***>. Découvrez pourquoi cela est important<https://aka.ms/LearnAboutSenderIdentification>

When I try to run your case, I get this:

 Starting FDS ...



 MPI Process      0 started on PN122745



 Reading FDS input file ...



WARNING: SPEC GAZ PYRO is not in the table of pre-defined species. Any unassigned SPEC variables in the input were assigned the properties of nitrogen.

WARNING: MATL AGGLOMERE, REAC 1. No product yields (NUs) set



ERROR(335): SURF PLANCHE cannot have a reacting MATL and IGNITION_TEMPERATURE. (CHID: TEST_B01)



ERROR: FDS was improperly set-up - FDS stopped (CHID: TEST_B01)

Please fix this (search user guide on IGNITION_TEMPERATURE) and try your case with the latest code. Thanks

**Comment by Eric-Ramboll** [comment]
The attached file has not been uploaded
[TEST_B02.txt](https://github.com/firemodels/fds/files/15416906/TEST_B02.txt)

**Comment by mcgratta** [comment]
```
&SURF ID='PLANCHE',
      RGB=210,180,140,
      BURN_AWAY=.TRUE.,
      BACKING='VOID',
      MATL_ID(1,1)='AGGLOMERE',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.0025 /
```
Why do you set `BACKING='VOID'`? This means that the back side boundary condition for each side of the plank is cold air. In this case, there is no need to set a `BACKING` because `'EXPOSED'` is the default and correct boundary condition.

**Comment by mcgratta** [comment]
One other thing. Your grid is coarse, in particular in the vertical direction. Temperatures are averaged over the area of the solid cell face, in which case you may not see high enough temperatures to ignite.

**Comment by Eric-Ramboll** [comment]
Several tests, following your advice. Despite a very fine mesh, in the area of my temperature sensors, no rise is detected during combustion.
However, I finally have a nice combustion, the disappearance of the elements over time.
I can't find any other syntax for my temperature sensors: any advice?
[Test_B04.txt](https://github.com/firemodels/fds/files/15444414/Test_B04.txt)

**Comment by mcgratta** [comment]
```
&DEVC ID='TÂ° + 1 mm', DEPTH=0.001, QUANTITY='INSIDE WALL TEMPERATURE', XYZ=0.00,0.00,0.01, IOR=3/
&DEVC ID='TÂ° + 2 mm', DEPTH=0.002, QUANTITY='INSIDE WALL TEMPERATURE', XYZ=0.00,0.00,0.01, IOR=3/
&DEVC ID='TÂ° + 5 mm', DEPTH=0.005, QUANTITY='INSIDE WALL TEMPERATURE', XYZ=0.00,0.00,0.01, IOR=3/
```
You have specified `IOR=3`. That means that FDS is going to embed the device in the nearest solid below the `XYZ` point. That is the 'Ignitor' which has no interior properties and is thus 20 C, the ambient temperature.

**Reply by Eric-Ramboll** [reply]
If I specifie IOR=-3, I have an error message
![image](https://github.com/firemodels/fds/assets/170494691/dc2f1937-c4cf-44f5-93e6-5009d0bdcff8)

According 22.2.2 my solid orientation is the positive z direction (from z=0.01 to z=0.35).
[Test_B04bis.txt](https://github.com/user-attachments/files/15510832/Test_B04bis.txt)

May I have an other temperature device synthaxe to get the temperature of insde cells in my solid, close the Ignitor ?

**Reply by mcgratta** [reply]
The convention used in FDS in this case would be `IOR=-3`. The orientation is the direction from the solid surface into the gas phase.  In your case, once the first grid cell burns away, your device will no longer be active. The device will not "follow" the burning front. I suggest that you apply these device on the side, pointing in the plus/minus 2 (y) direction.

**Comment by Eric-Ramboll** [comment]
OK, I understand now. Thank you for your advice.
